source/assets/Components/CookingMode.js

```diff
@@ -21,49 +21,92 @@ class CookingMode extends HTMLElement {
       background-size: cover;
       padding: 23.5px;
       color: white;
+      text-align: center;
     }
     button {
+      min-width: 10%;
+      border-radius: 18px;
       background-color: white;
       border: 1.5px solid #ca676a;
+      text-align: center;
+      font-size: 14pt;
+      color:#ca676a;
+    }
+    #btnNav {
+      position: fixed;
+      bottom: 40pt;
+    }
+    #btnNav button {
+      height: 22pt;
+      width: 22pt;
+      border-radius: 50%;
+      background-color: white;
+      border: 1.5px solid #ca676a;
+      text-align: center;
+      font-size: 14pt;
+      color:#ca676a;
+    }
+    .css-outer {
+      display: flex;
+      justify-content: center;
+    }
+    .special-outer {
+      display: table;
+      position: absolute;
+      top: 0;
+      left: 0;
+      height: 100%;
+      width: 100%;
+      text-align: center;
+    }
+    
+    .middle {
+      display: table-cell;
+      vertical-align: middle;
+      text-align: center;
+    }
+    
+    #cooking-steps {
+      margin-left: auto;
+      margin-right: auto;
+      width: 100%;
+      text-align: center;
+      font-size: 18pt;
+    }
+    #cooking-timer {
+      position: fixed;
+      bottom: 80pt;
+      text-align: center;
     }
         `;
     article.innerHTML = `
         <h2>Cooking Mode</h2>
+
+        <div class="css-outer">
+        <div class="special-outer">
+        <div class="middle">
+        <button>Quit</button>
         <!--Cooking Steps-->
         <div id="cooking-steps">
         </div>
+        </div>
+        </div>
+
         <div id="btnNav">
         </div>
+        </div>
+
+        <div class="css-outer">
         <!--Cooking Timer-->
         <div id="cooking-timer">
         </div>
-    
-        <!--Back to the Recipe Page-->
-        <div class="back-to-recipe">
-            <p>Back to the Home Page</p>
-            <button><a href="home.html"> LEAVE </a></button>
         </div>
+    
+
         `;
 
     // Append elements to the shadow root
     this.shadowRoot.append(styles, article);
-
-    /*function convertHMS(value) {
-      const sec = parseInt(value, 10); // convert value to number if it's string

```

Comment:
Can you remove this code if you don't think it'll be used

Reply:
oh, they are useful functions which are called later